packages/core/src/Project.ts

```diff
@@ -148,6 +149,17 @@ export class Project {
     return this.size;
   }
 
+  public setVariables(variables: object) {
+    this.variables = variables;
+    if (this.scenes.current[0]) {
+      this.scenes.current[0].variables.updateSignals();
+    }
```

Comment:
It should be:
```suggestion
    this.currentScene.current?.variables.updateSignals();
```
There can be more than one scene in a project. We're interested in updating the current one, not the first one.